internal/provider/resource_docker_container.go

```diff
@@ -64,6 +64,20 @@ func resourceDockerContainer() *schema.Resource {
 				Optional:    true,
 			},
 
+			"wait": {
+				Type:        schema.TypeBool,
+				Description: "If `true`, then the Docker container is waited for being healthy state after creation. If `false`, then the container health state is not checked. Defaults to `false`.",
```

Comment:
Maybe mention, that even with `false`, the container will be spun up, but terraform exists without waiting.